src/js/profile/mobile/widget/Calendar.js

```diff
@@ -110,22 +113,30 @@
 						day = day - 1;
 						leftDays = leftDays - 1;
 						cell = row.insertCell();
-						cell.innerHTML = prevLastDate - day;
-						cell.style.opacity = "10%";
+						cell.insertAdjacentHTML("afterbegin", divElement);
+						cell.firstChild.innerHTML = prevLastDate - day;
+						cell.firstChild.style.opacity = "10%";
 					}
 					while (leftDays != 0) {
 						if (setDays > prevLastDate) { // next disabled days.
 							cell = row.insertCell();
-							cell.innerHTML = nextMonthDate;
-							cell.style.opacity = "40%";
+							cell.insertAdjacentHTML("afterbegin", divElement);
+							cell.firstChild.innerHTML = nextMonthDate;
+							cell.firstChild.style.opacity = "40%";
```

Comment:
I think is better to use css class instead of inline style

Reply:
done.